generators/app/tasks/appSetup/androidProjectSetup.js

```diff
@@ -2,7 +2,7 @@ function addDotenvPluginAndRNScreen() {
   const buildGradleContent = this.fs.read(`${this.projectName}/android/app/build.gradle`);
   let updatedBuildGradleContent = buildGradleContent.replace(
     'apply plugin: "com.android.application"',
-    'apply plugin: "com.android.application"\napply from: project(\':react-native-config\').projectDir.getPath() + "/dotenv.gradle"'
+    'apply plugin: "com.android.application"\nproject.ext.envConfigFiles = [qadebug: ".env.dev",\nqarelease: ".env.dev",\nstagedebug: ".env.stage",\nstagerelease: ".env.stage",\nproductiondebug: ".env.production",\nproductionrelease: ".env.production",\nanothercustombuild: ".env"]\napply from: project(\':react-native-config\').projectDir.getPath() + "/dotenv.gradle"'
```

Comment:
Fix indentation and missing `;` at the end of the Array. Also, remove `anothercustombuild`.

<img width="883" alt="Captura de Pantalla 2020-04-02 a la(s) 12 58 28" src="https://user-images.githubusercontent.com/8583214/78271696-da2c7900-74e2-11ea-9569-adbf96c4ba9b.png">